### PR TITLE
Switch tests to D9.

### PR DESCRIPTION
**Pull request by kporras07**

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -127,7 +127,7 @@ jobs:
             - run:
                 name: Test D8 with BitBucket
```
Comment (jspellman814) [inline .circleci/config.yml]:
```suggestion
                name: Test D9 with BitBucket
```

**Comment by jspellman814** [comment]
@kporras07 Couple things I noticed on unchanged lines that I can't comment on:
Line 136 needs change to: Clean up D9 Bitbucket fixtures after pass
Line 139 needs change to: Clean up D9 Bitbucket fixtures after pass

And a couple typos I noticed unrelated to your changes:
Line 219 needs change to: Test WordPress with BitBucket
Line 227 needs change to: Clean up WordPress Bitbucket fixtures after pass
Line 230 needs change to: Clean up WordPress Bitbucket fixtures after pass

**Comment by kporras07** [comment]
@jspellman814 thanks. Fixed

File: .circleci/config.yml [hunk 2]
```diff
@@ -216,18 +216,18 @@ jobs:
                 name: Disable strict SSH host checking to prevent SSH connect issues
                 command: echo "StrictHostKeyChecking no" >> "$HOME/.ssh/config"
             - run:
-                name: Test D8 with BitBucket
+                name: Test Wordpressd with BitBucket
```
Comment (jspellman814) [inline .circleci/config.yml]:
```suggestion
                name: Test WordPress with BitBucket
```

File: .circleci/config.yml [hunk 3]
```diff
@@ -216,18 +216,18 @@ jobs:
...
                 command: echo "BitBucket test skipped"
                 #command: ./.circleci/test-bitbucket-repo.sh "pantheon-systems/example-wordpress-composer:dev-master"
             - run:
                 name: Watch CircleCI tests
                 command: echo "Skipping the watch-cicle step too"
                 #command: ./.circleci/watch-circle.sh
             - run:
-                name: Clean up D8 Bitbucket fixtures after pass
+                name: Clean up Wordpress Bitbucket fixtures after pass
```
Comment (jspellman814) [inline .circleci/config.yml]:
```suggestion
                name: Clean up WordPress Bitbucket fixtures after pass
```

File: .circleci/config.yml [hunk 4]
```diff
@@ -216,18 +216,18 @@ jobs:
...
                 command: ./.circleci/cleanup-fixtures.sh ok
             - run:
-                name: Clean up D8 Bitbucket fixtures after failure
+                name: Clean up Wordpress Bitbucket fixtures after failure
```
Comment (jspellman814) [inline .circleci/config.yml]:
```suggestion
                name: Clean up WordPress Bitbucket fixtures after failure
```

**Review by jspellman814 — APPROVED** [review]

**Pull request merged**